apps/language_models/scripts/vicuna.py

```diff
@@ -114,11 +112,18 @@
     "--cache_vicunas",
     default=False,
     action=argparse.BooleanOptionalAction,
-    help="For debugging purposes, creates a first_{precision}.mlir and second_{precision}.mlir and stores on disk"
+    help="For debugging purposes, creates a first_{precision}.mlir and second_{precision}.mlir and stores on disk",
 )
 
 
-def brevitas〇matmul_rhs_group_quant〡shape(lhs: List[int], rhs: List[int], rhs_scale: List[int], rhs_zero_point: List[int], rhs_bit_width: int, rhs_group_size: int) -> List[int]:
```

Comment:
pls revert the format

Reply:
reverted in https://github.com/nod-ai/SHARK/pull/1702/commits/7c35d276c4a14e3a52de8c09b8a681cc86cc9777